.github/ISSUE_TEMPLATE/q_and_a.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Question and answer
+about: Ask your questions
+title: ''
+labels: question
```

Comment:
We now have two labels: 'question' and 'help wanted'. Aren't these the same?

Reply:
thanks for pointing this out. Yes, they are the same. I should edit the default list of the labels in this repo.